### PR TITLE
chore/feedback: use anonymous records; other feedback updates

### LINKED ISSUE #2: Feedback
- [x] remove `Compacted` event for now
- [ ] change `LeaseEvent.getEventId` to `Lease.tryGetEventId`
- [ ] change `StreamState.events` to `StreamState.effectiveEvents`
- [x] inline type aliases
- [x] change naming of apply, evolve
- [x] use anonymous records

### PR DESCRIPTION
**Pull request by andymeierdev**
closes #2

### CODE REVIEW

**Comment by bartelink** [comment]
2.0.0-preview1 is being indexed as we speak; should be relatively minor compared to what you are presently targeting

File: src/Lease/Lease.Aggregate.fs [hunk 1]
```diff
@@ -8,57 +8,17 @@ type ObservationDate =
...
-     -> 'View
-
-type Query<'EntityId,'DomainEvent,'View> = 
-    'EntityId
-     -> ObservationDate 
-     -> Projection<'DomainEvent,'View>
-     -> AsyncResult<'View,string>
-
-type Aggregate<'DomainCommand,'DomainEvent,'DomainState> =
+type Aggregate =
```
Comment (bartelink) [inline src/Lease/Lease.Aggregate.fs:11]: What about just doing a normal actual `type` ?

```
type Aggregate(name) =
    member __.Entity = name
    member __.Initial = initial
    member __.EvolveDomain: LeaseState -> LeaseEvent -> LeaseState = evolveDomain?
```

(NB I'm not sure what this type is even for (and I'm not sure exposing all those things is necessarily useful, seems like an ISP violation to me) - my one and only point is that I dont get the purpose of putting functions in a record - I may be underinformed and/or missing something)

Comment (andymeierdev) [reply src/Lease/Lease.Aggregate.fs:11]: I had seen this done somewhere and thought it looked nice so just adopted it. I do think the classes are cleaner for exposing only the functions you need. will refactor on the next upate

**Comment by andymeierdev** [comment]
closing in favor of #5

**Pull request closed without merging**